slsDetectorServers/slsDetectorServer/slsDetectorFunctionList.h

```diff
@@ -252,8 +252,10 @@ int 		setDetectorPosition(int pos[]);
 #ifdef EIGERD
 int			setQuad(int value);
 int			getQuad();
-int		setInterruptSubframe(int value);
+int			setInterruptSubframe(int value);
 int			getInterruptSubframe();
+int 		setReadNLines(int value);
```

Comment:
Is it only on github the formatting is off?

Reply:
It doesnt look so on my editor, but I haven't run clang on any of the servers anyway. Will do the formatting at a later step.